rfcs/convergence/authoring-stories.md

```diff
@@ -0,0 +1,261 @@
+# RFC: Authoring storybook stories
+
+---
+
+_List contributors to the proposal:_ @hotell
+
+<!-- toc -->
+
+- [Summary](#summary)
+- [Problem statement](#problem-statement)
+- [Detailed Design or Proposal](#detailed-design-or-proposal)
+  - [1. what should we use for interactive props playground - controls or knobs?](#1-what-should-we-use-for-interactive-props-playground---controls-or-knobs)
+  - [2. how to use `argTypes`](#2-how-to-use-argtypes)
+  - [3. should controls work for all stories or only for general/default one](#3-should-controls-work-for-all-stories-or-only-for-generaldefault-one)
+  - [4. should we render descriptions table in Canvas Control pane](#4-should-we-render-descriptions-table-in-canvas-control-pane)
+  - [Pros and Cons](#pros-and-cons)
+- [Discarded Solutions](#discarded-solutions)
+- [Open Issues](#open-issues)
+
+<!-- tocstop -->
+
+## Summary
+
+Currently we have no style-guide/functionality requirements how to be consistent when writing stories for converged components. This RFC should describe (as we progress) the style we wanna go forward.
+
+## Problem statement
+
+For convergence (vNext) we agreed on collocated stories approach that uses [Component Story Format (CSF)](https://storybook.js.org/docs/react/api/csf) for implementation.
+
+**What are we missing:**
+
+1. what should we use for interactive props playground - controls or knobs?
+2. how to use `argTypes`
+3. should controls work for all stories or only for general/default one
+4. should we render descriptions table in Canvas Control pane
+5. how to author e2e suites for those stories (in written form)
+6. how to properly annotate stories with TS metadata to get best DX possible
+
+> 💡 NOTE: This RFC will address first 3 points for now (should be updated with others later).
```

Comment:
Does this mean that the proposals for the other points will be added in this PR or as a part of a follow up PR ?

Reply:
yeah follow-up (living document)